### PR TITLE
docs: Add documentation for the multi-ecosystems-lookup work

### PR DESCRIPTION
**Pull request by lucasamonrc**

### CODE REVIEW

File: docs/reference/services/connect-service.md [hunk 1]
```diff
@@ -155,11 +157,11 @@ Once a Session has been [created](#create-session), the verification flow must b
...
 
 Use the [Get Session](#get-session) call to get the current Session object, including its {{ field_ref("GetSessionResponse", "session.state") }} and {{ field_ref("GetSessionResponse", "session.result_vp") }}.
 
-The verification data is in the form of a [Verifiable Presentation <small>:material-open-in-new:</small>](https://www.w3.org/TR/vc-data-model/#presentations){target=_blank}, 
+The verification data is in the form of a [Verifiable Presentation <small>:material-open-in-new:</small>](https://www.w3.org/TR/vc-data-model/#presentations){target=_blank},
 which will contain one [Verifiable Credential <small>:material-open-in-new:</small>](/learn/concepts/credentials){target=_blank} for each Verification requested during [Session Creation](#create-session).
 
 Below is an example JSON document representing the structure of the output of a Session created with a single Verification of type `GOVERNMENT_ID`:
```
Comment (janpieterz) [inline docs/reference/services/connect-service.md:167]: We probably want to showcase the complete response and, notably, the `result_vp` has become a lot less important (it'll be different for each credential, so we can't push e.g. a "givenName" field.

File: docs/reference/services/connect-service.md [hunk 2]
```diff
@@ -155,11 +157,11 @@ Once a Session has been [created](#create-session), the verification flow must b
 
 ## Get Verification Results
 
-If a verification is successful, its {{ field_ref("GetSessionResponse", "session.state") }}  will be {{ field_ref("GetSessionResponse", "session.state.IDV_SUCCESS") }}, and the resultant data will be available as a JSON string in the {{ field_ref("GetSessionResponse", "session.result_vp") }} field. 
+If a verification is successful, its {{ field_ref("GetSessionResponse", "session.state") }}  will be {{ field_ref("GetSessionResponse", "session.state.IDV_SUCCESS") }}, and the resultant data will be available as a JSON string in the {{ field_ref("GetSessionResponse", "session.result_vp") }} field.
```
Comment (janpieterz) [inline docs/reference/services/connect-service.md:160]: We should add a strong note about the `result_vp` containing different proofs depending on the issuer, where-as the normalized data _does not_. We don't really want people to use the `result_vp` unless they're absolutely sure what they're doing.

File: docs/reference/services/connect-service.md [hunk 3]
```diff
@@ -65,6 +65,8 @@ Fetch an identity verification session by its ID.
 
 If the Session has been completed successfully, the results will be available in the {{ field_ref("GetSessionResponse", "session.result_vp") }} field of the returned Session. See [Get Verification Results](#get-verification-results) for information on parsing this field.
```
Comment (janpieterz) [inline docs/reference/services/connect-service.md:66]: We should flip this and add a bit about how the `result_vp` will be different depending on issuer as well as how it's better to use the normalized data

**Review by janpieterz — CHANGES_REQUESTED** [review]
Let's change the focus from `result_vp` to normalized data parts and highlight the painpoints of `result_vp` usage more/better + add structure of normalized data.

**Comment by lucasamonrc** [comment]
I'm closing, we will revisit the docs this following week in preparation for our launch.

**Comment by lucasamonrc** [comment]
Closing this, we will address all of these points when we prepare our docs for the product launch in the next week.

**Pull request closed without merging**